CI/build-deps-windows-native.ps1

```diff
@@ -0,0 +1,146 @@
+    [Switch]$Quiet = $(if (Test-Path variable:Quiet) { $Quiet }),
+    [Switch]$Verbose = $(if (Test-Path variable:Verbose) { $Verbose }),
+    [Switch]$NoChoco = $(if (Test-Path variable:NoChoco) { $NoChoco }),
+    [Switch]$SkipDependencyChecks = $(if (Test-Path variable:SkipDependencyChecks) { $SkipDependencyChecks }),
+    [String]$BuildDirectory = "build",
+    [ValidateSet("32-bit", "64-bit")]
+    [String]$BuildArch = (Get-CimInstance CIM_OperatingSystem).OSArchitecture,
+    [ValidateSet("Release", "RelWithDebInfo", "MinSizeRel", "Debug")]
+    [String]$BuildConfiguration = "RelWithDebInfo"
+)
+
+##############################################################################
+# Windows OBS deps native build script
+##############################################################################
+#
+# This script contains all steps necessary to:
+#
+#   * Build OBS required dependencies
+#   * Create 64-bit and 32-bit variants
+#
+# Parameters:
+#   -Help                   : Print usage help
+#   -Quiet                  : Suppress most build process output
+#   -Verbose                : Enable more verbose build process output
+#   -SkipDependencyChecks   : Skip dependency checks
+#   -NoChoco                : Skip automatic dependency installation
+#                             via Chocolatey
+#   -BuildDirectory         : Directory to use for builds
+#                             Default: Win64 on 64-bit systems
+#                                      Win32 on 32-bit systems
+#   -BuildArch              : Build architecture to use (32-bit or 64-bit)
+#   -BuildConfiguration     : Build configuration to use
+#                             Default: RelWithDebInfo
+#
+##############################################################################
+
+$ErrorActionPreference = "Stop"
+
+$_RunObsDepsBuildScript = $true
+$ProductName = "obs-deps"
+
+$CheckoutDir = git rev-parse --show-toplevel
+$DepsBuildDir = "${CheckoutDir}\windows_native_build_temp"
+
+. ${CheckoutDir}\CI\include\build_support_windows.ps1
+
+$ObsBuildDependencies = @(
+    @('mbedtls', '523f0554b6cdc7ace5d360885c3f5bbcc73ec0e8'),
+    @('cmocka', '9c114ac31a33217cf003bbb674c1aff7bb048917'),
+    @('freetype', '6a2b3e4007e794bfc6c91030d0ed987f925164a8'),
+    @('curl', '315ee3fe75dade912b48a21ceec9ccda0230d937'),
+    @('rnnoise', '90ec41ef659fd82cfec2103e9bb7fc235e9ea66c'),
+    @('speexdsp', '20ed3452074664ad07e380e51321b148acebdf20'),
+    @('detours', 'e4bfd6b03e50de46b47abfbd1e46b384f0c5f833'),
+    @('luajit', '0bf80b07b0672ce874feedcc777afe1b791ccb5a'),
+    @('python', '3.6.2'),
+    @('vulkan', '1.2.131.2', '49d515f091d69c005a9e52e69829d28383fe157a764caebaa3dbc4b8d9bb383d'),
+    @('nasm', '2.15.01'),
+    @('swig', '3.0.12', '21ce6cbe297a56b697ef6e7e92a83e75ca41dedc87e48282ab444591986c35f5')
+)
+
+function Build-OBS-Deps-Main {
+    Ensure-Directory "${CheckoutDir}"
+    Write-Step "Fetching version tags..."
+    & git fetch origin --tags
```

Comment:
I'm sort of leaning towards keeping it so that we can see the output in the GHA log to make sure that tags were indeed fetched.  Unless there is a concern that this may emit an error and therefore break the script?